shared/js/ui/base/ui-wrapper.es6.js

```diff
@@ -55,13 +55,9 @@ const openExtensionPage = (path) => {
     browser.tabs.create({ url: getExtensionURL(path) })
 }
 
-const openOptionsPage = (browser) => {
-    if (browser === 'moz') {
```

Comment:
I wonder what the purpose of this different code path was for Firefox. Seems weird we can just remove it?